lib/sober_swag/controller.rb

```diff
@@ -40,28 +45,40 @@ module Types
       #     PostsController::Show # the container module for everything in this action
       #     PostsController::Show::PathParams # the dry-struct type for the path attribute.
       # So, in the same controller, you can refer to Show::PathParams to get the type created by the 'path_params' block above.
+      #
```

Comment:
Lines 43-47 have some parsing issues, or are missing markdown styles.

![image](https://user-images.githubusercontent.com/3358255/108797751-31976580-7549-11eb-97fc-3a4f0875095e.png)

Reply:
607a641

Reply:
What do we think of using the `@!parse [language]` YARD tag as opposed to triple backticks?

https://rubydoc.info/gems/yard/file/docs/Tags.md#parse

```rb
# @!parse [c]
#   void Init_Foo() {
#     rb_define_method(rb_cFoo, "method", method, 0);
#   }
```